samples/sample_encode/src/sample_encode.cpp

```diff
@@ -112,6 +112,7 @@ void PrintHelp(msdk_char *strAppName, const msdk_char *strErrorMessage, ...)
 #if (MFX_VERSION >= 1027)
     msdk_printf(MSDK_STRING("   [-round_offset_in file]  - use this file to set per frame inter/intra rounding offset(for AVC only)\n"));
 #endif
+    msdk_printf(MSDK_STRING("   [-repackctrl file]       - file to input max encoded frame size,number of pass and delta qp for each frame(ENCODE only)\n"));
```

Comment:
...size, number of passes and delta qp for each pass ...

Reply:
Will modify it, thanks!